packages/trader/src/Modules/MT5/Components/mt5-real-account-display.jsx

```diff
@@ -241,7 +254,7 @@ const MT5RealAccountDisplay = ({
             descriptor={localize(
                 'Trade CFDs on forex, stocks & indices, commodities, and cryptocurrencies with leverage.'
             )}
-            specs={should_show_eu ? eu_real_financial_specs : real_financial_specs}
+            specs={financial_specs}
```

Comment:
Should we also update compare accounts modal content to show different specs for Australian clients?

Reply:
@balakrishna-binary Yes, I think so. I updated the content of that table too. thanks